### ISSUE TITLE
Funding post - anonymous donor

### ISSUE THREAD

**Issue by krrisis** [Giveth/Communication#124]
Content: 

- Giveth is extremely grateful to have received a late Christmas gift, a last 2018 gift to make 2019 even better -- we received 285 eth (40K+ DAI)

- we need more supporters to build the Future of Giving. FUG is not only the DApp, but the entire Galaxy (summary of text blocks on giveth.io )

- mention briefly that we now have two main entry points, the unicorn dac or the giveth dac, explain briefly that one is more 'top-down', one is a more bottom up approach (few sentences + link to posts)

- How can you support? Summarize the content found on giveth.io/join and link to it 

Tone of voice: 
- Keep it warm, open, embracing (in line with our Giveth Manifesto)
- clear & to the point, a handy guidance to donate to giveth

Notes
- To be published on the giveth medium 
- More info & feedback via Kris (might even write this myself, tbd)
- deadline final version: 10/01 at the latest?

**Comment by krrisis** [comment]
@GriffGreen add any comments here :) 

@elietzioni this should definitely also go in here - a reference on how we decided in our decentralized way what to do with the funds! https://www.loomio.org/d/Sbn1FeT7/285-eth-what-am-i-gonna-do-

**Comment by GriffGreen** [comment]
Yep! We did the loomio and then we talked about it in the Community meeting but sadly it had a recording issue :-P so it didnt really record anything useful :-(

the final verdict was what was said in the loomio, but its interesting... at least i think its interesting.... that we had to do a whole process around "what do we do with this donation" because it was sent to the multisig... but if the donor would have sent it into the dapp there wouldn't have been any governance needed because that is handled by the dapp :-D If they would have just donated to a specific campaign, or to a DAC, then it would have been clear who decides what to do with those funds...

In our case, we decided to send it 50/50 to the Giveth DApp to cover some larger expenses and the rest to the Unicorn DAC to support our decentralized 

link to the Unicorn DAC and talk about how it works a little bit.. basically, Giveth Unicorns that have gone thru the onboarding process get to use their share of the donation to steer the direction of the Giveth Galaxy

Relevant links
https://wiki.giveth.io/dac/Unicorn-DAC/
https://wiki.giveth.io/dac/Unicorn-DAC-getfunded/
https://wiki.giveth.io/policy/Unicorn-onboarding/

**Comment by geleeroyale** [comment]
I think this is a good opportunity to inform people about these choices! We should change our official policy in a way that everything donated to Giveth DAC will be routed according to our community's decision, but if donors want to support something specific within Giveth, it will be that campaigns/community's decision - We should communicate this loudly and clearly, as I think we are the only body that accepts donations in this way currently!

So to summarize - we should be more clear on giveth-io/donate about possibilities and who decides on routing donations, as well as take the change of this donation to explain with lots of pictures how donations in Giveth (with or without DApp) work!

**Comment by krrisis** [comment]
Good feedback Kay! Have added a paragraph specifically on this. 

@GriffGreen and @mroberts10 Eli has already spent quite some time on this, but this is ofc an important one, as it is a first ask for funding: could you briefly check if the current version is clear to you and is good to go? thx! https://docs.google.com/document/d/1lhE8D58GiowlJwQUNFOuOpgengxDuYOtSN-IQmQ88rw/edit?usp=sharing
